modules/nodes-nodes-managing-about.adoc

```diff
@@ -12,43 +12,52 @@ To make configuration changes to a cluster, or MachinePool, you must create a Cu
 . Obtain the label associated with the static CRD, Machine Config Pool, for the type of node you want to configure.
 Perform one of the following steps:
 
-.. Edit the machineconfigpool master, add label"custom-kubelet: small-pods"
+.. Check current labels of the desired machineconfigpool.
 +
-For example:
+For example
 +
 ----
-$ oc edit machineconfigpool worker
+$  oc get machineconfigpool  --show-labels 
+NAME      CONFIG                                             UPDATED   UPDATING   DEGRADED   LABELS
+master    rendered-master-e05b81f5ca4db1d249a1bf32f9ec24fd   True      False      False      operator.machineconfiguration.openshift.io/required-for-upgrade=
+worker    rendered-worker-f50e78e1bc06d8e82327763145bfcf62   True      False      False      
+----
 
-metadata:
-  creationTimestamp: 2019-01-31T07:10:04Z
-  generation: 3
-  labels:
-    custom-kubelet: small-pods <1>
-    operator.machineconfiguration.openshift.io/required-for-upgrade: ""
+.. Add a custom label to the desired machineconfigpool.
++
+For example:
++
+----
+$ oc label machineconfigpool worker custom-kubelet=enabled
 ----
-<1> If a label has been added it appears under `labels`.
 
-.. If the label is not present, add a key/value pair under `labels`.
 
-. Create a Custom Resource (CR) for your configuration change.
+. Create a KubeletConfig Custom Resource (CR) for your configuration change.
 +
 For example:
 +
-.Sample configuration for a *max-pods* CR
+.Sample configuration for a *custom-config* CR
 ----
 apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfigPool
+kind: KubeletConfig
 metadata:
-  name: set-max-pods <1>
+  name: custom-config <1>
```

Comment:
Does the name need to be `custom-config` or can the user apply any name? I think I was trying to create a descriptive name for the CR to identify what the CR is doing.

Reply:
Yes, the user can apply any name. But the example CR is no longer about configuring max pods in the kubelet. Now several options are modified in the provided example.